### PR TITLE
python-whoosh: pip install `wheel` in %check section to enable ptest

### PR DESCRIPTION
**Pull request by mfrw**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Fix ptest build. All tests pass!

###### Change Log  

- python-whoosh: pip install `wheel` in %check section to enable ptest
- python-whoosh: switch to github tarball instead of pypi

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- NA

###### Test Methodology

- Local Build with RUN_CHECK=y

### CODE REVIEW

File: SPECS-EXTENDED/python-whoosh/python-whoosh.spec [hunk 1]
```diff
@@ -62,6 +62,7 @@ rm -rf docs/html/.doctrees
 %py3_install
 
 %check
+%{__python3} -m pip install wheel
```
Comment (PawelWMS) [inline SPECS-EXTENDED/python-whoosh/python-whoosh.spec:65]: Wouldn't this be enough?

```suggestion
pip3 install wheel
```

File: SPECS-EXTENDED/python-whoosh/python-whoosh.spec [hunk 2]
```diff
@@ -1,18 +1,18 @@
 # For bootstrapping sphinxcontrib-websupport
 %bcond_without docs
 
-%global mod_name Whoosh
+%global mod_name whoosh
```
Comment (PawelWMS) [inline SPECS-EXTENDED/python-whoosh/python-whoosh.spec:4]: I think we don't need this anymore.

**Review by PawelWMS — APPROVED** [review]
Just 2 short comments/requests but no blockers.

**Review by anphel31 — APPROVED** [review]

File: SPECS-EXTENDED/python-whoosh/python-whoosh.spec (hunk 1 above)
Comment (mfrw) [reply SPECS-EXTENDED/python-whoosh/python-whoosh.spec:65]: Yes, I actually read a blog post: [Why should one use python -m pip](https://snarky.ca/why-you-should-use-python-m-pip/), which convinced me to use `python -m pip install ....`.

**Pull request merged**